sphinx/source/glossary.rst

```diff
@@ -7,7 +7,7 @@ Glossary
     `Azure Confidential Compute <https://azure.microsoft.com/en-us/solutions/confidential-compute/>`_ protects the confidentiality and integrity of your data and code while it’s processed in the public cloud.
 
   Consensus Protocols
-    The term Consensus protocol refers to either Raft or PBFT. Generic Consensus terminology will use `primary` node and `backup` node to indicate node responsibility in carrying out the protocol(s). These correspond in Raft to `leader` and `follower`.
+    The term Consensus protocol refers to either Raft or PBFT. Generic Consensus terminology will use `primary` node and `backup` node to indicate node responsibility in carrying out the protocol(s). These correspond in Raft to `leader` and `follower`. More information about the consensus protocols can be found :ref:`here <developers/consensus:Consensus Protocols>`.
```

Comment:
I am not sure we want to get into educating users of computer science concepts. Maybe link to the Wikipedia page (https://en.wikipedia.org/wiki/Consensus_(computer_science)) it provides a good overview

Reply:
Maybe put PBFT and Raft into the glossary?